File: RxTools-library/src/main/java/com/vondear/rxtools/view/wheelhorizontal/OnWheelScrollListener.java
package com.vondear.rxtools.view.wheelhorizontal;


/**
 * @author vondear
 * Wheel scrolled listener interface.
 */
public interface OnWheelScrollListener {
	/**
	 * Callback method to be invoked when scrolling started.
	 * @param wheel the spinnerwheel view whose state has changed.
	 */
	void onScrollingStarted(AbstractWheel wheel);
	
	/**
	 * Callback method to be invoked when scrolling ended.
	 * @param wheel the spinnerwheel view whose state has changed.
	 */
	void onScrollingFinished(AbstractWheel wheel);
}

File: RxUI/src/main/java/com/tamsiree/rxui/view/dialog/wheel/OnWheelScrollListener.kt
package com.tamsiree.rxui.view.dialog.wheel

/**
 * @author tamsiree
 * Wheel scrolled listener interface.
 */
interface OnWheelScrollListener {
    /**
     * Callback method to be invoked when scrolling started.
     * @param wheel the wheel view whose state has changed.
     */
    fun onScrollingStarted(wheel: WheelView?)

    /**
     * Callback method to be invoked when scrolling ended.
     * @param wheel the wheel view whose state has changed.
     */
    fun onScrollingFinished(wheel: WheelView?)
}